CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+Thanks you for your interest in contributing to Microsoft Caboodle! In this document we'll outline what you need to know about contributing and how to get started.
+
+## Code of Conduct
+
+Please see our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Prerequisite
+
+You will need to complete a Contribution License Agreement before any pull request can be accepted. Complete the CLA at https://cla2.dotnetfoundation.org/.
+
+## Contributing Code
+
+Check out [A Beginner's Guide for Contributing to Microsoft Caboodle](https://blog.xamarin.com/beginners-guide-contributing-xamarin-forms/).
+
+### Bug Fixes
+
+If you're looking for something to fix, please browse [open issues](https://github.com/xamarin/Caboodle/issues). 
+
+Follow the style used by the [.NET Foundation](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), with two primary exceptions:
+
+- We do not use the `private` keyword as it is the default accessibility level in C#.
+- We use hard tabs over spaces. You can change this setting in VS 2015 via `Tools > Options` and navigating to `Text Editor > C#` and selecting the "Keep tabs" radio option.
```

Comment:
What's everyone's opinion here? I like spaces over tabs personally.

Reply:
I used to be a spaces person, but tabs are nice as you can adjust the indenting size based on your own preferences.

Reply:
i think the default for VS and VS4M is tabs isn't it?

Reply:
This should say VS 2017 as that is the only ide we support on windows

Reply:
I believe the default is spaces and we should go with that.